PROJECT

```diff
@@ -1,16 +1,16 @@
-domain: onmetal.de
+domain: ironcore.dev
 layout:
 - go.kubebuilder.io/v3
-projectName: cephlet
-repo: github.com/onmetal/cephlet
+projectName: ceph-provider
+repo: github.com/ironcore-dev/ceph-provider
 resources:
 - controller: true
-  domain: onmetal.de
-  group: storage.api.onmetal.de
+  domain: ironcore.dev
+  group: storage.ironcore.dev
   kind: Volume
   version: v1alpha1
 - controller: true
-  domain: onmetal.de
+  domain: ironcore.dev
   group: storage.api
```

Comment:
```suggestion
  group: storage.ironcore.dev
```

Reply:
If I got it correct, the group should not include the full domain: https://kubebuilder.io/reference/project-config